tests/notebooks/scRNA_and_smFISH.ipynb

```diff
@@ -496,7 +487,8 @@
    "outputs": [],
    "source": [
     "from sklearn.manifold import TSNE\n",
-    "def get_common_t_sne(latent_seq, latent_fish, n_samples=1000):\n",
+    "n_samples_tsne = 1000 if n_samples_tsne is None else n_samples_tsne\n",
```

Comment:
I think it would be better just to have `n_samples_tsne = 1000` here (without checking for None values). We can still find and replace the code in the unit tests. (Use `\d+` in a regex to match any digits)